resources/icons/16px/british-pound.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M7.00009 5C7.00009 4.73736 7.05182 4.47729 7.15233 4.23463C7.25284 3.99198 7.40016 3.7715 7.58588 3.58579C7.77159 3.40007 7.99207 3.25275 8.23472 3.15224C8.47737 3.05173 8.73745 3 9.00009 3C9.26273 3 9.52281 3.05173 9.76546 3.15224C10.0081 3.25275 10.2286 3.40007 10.4143 3.58579C10.6 3.7715 10.7473 3.99198 10.8478 4.23463L12.6956 3.46927C12.4946 2.98396 12.2 2.54301 11.8285 2.17157C11.4571 1.80014 11.0161 1.5055 10.5308 1.30448C10.0455 1.10346 9.52538 1 9.00009 1C8.4748 1 7.95466 1.10346 7.46936 1.30448C6.98405 1.5055 6.5431 1.80014 6.17166 2.17157C5.80023 2.54301 5.50559 2.98396 5.30457 3.46927C5.10355 3.95457 5 4.47471 5 5V7H4C3.44772 7 3 7.44772 3 8C3 8.55228 3.44772 9 4 9H5V10.5C5 11.6331 4.77099 12.2907 4.54295 12.6328C4.3327 12.9482 4.12995 13 4 13H3V15H13V13H6.591C6.86122 12.3158 7 11.4782 7 10.5V9H10C10.5523 9 11 8.55229 11 8C11 7.44772 10.5523 7 10 7H7L7.00009 5Z" fill="black"/>
+</svg>
```

Comment:
Can we omit the `width`, `height`, and `fill` properties on the svg and path elements? They aren't necessary:
```svg
<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
<path d="M7.00009 5C7.00009 4.73736 7.05182 4.47729 7.15233 4.23463C7.25284 3.99198 7.40016 3.7715 7.58588 3.58579C7.77159 3.40007 7.99207 3.25275 8.23472 3.15224C8.47737 3.05173 8.73745 3 9.00009 3C9.26273 3 9.52281 3.05173 9.76546 3.15224C10.0081 3.25275 10.2286 3.40007 10.4143 3.58579C10.6 3.7715 10.7473 3.99198 10.8478 4.23463L12.6956 3.46927C12.4946 2.98396 12.2 2.54301 11.8285 2.17157C11.4571 1.80014 11.0161 1.5055 10.5308 1.30448C10.0455 1.10346 9.52538 1 9.00009 1C8.4748 1 7.95466 1.10346 7.46936 1.30448C6.98405 1.5055 6.5431 1.80014 6.17166 2.17157C5.80023 2.54301 5.50559 2.98396 5.30457 3.46927C5.10355 3.95457 5 4.47471 5 5V7H4C3.44772 7 3 7.44772 3 8C3 8.55228 3.44772 9 4 9H5V10.5C5 11.6331 4.77099 12.2907 4.54295 12.6328C4.3327 12.9482 4.12995 13 4 13H3V15H13V13H6.591C6.86122 12.3158 7 11.4782 7 10.5V9H10C10.5523 9 11 8.55229 11 8C11 7.44772 10.5523 7 10 7H7L7.00009 5Z"/>
</svg>
```
<img src="https://github.com/user-attachments/assets/0720625b-1c8c-4012-9788-7412f3a9ad80" alt="british-pound" width="16" />